tally.py

```diff
@@ -0,0 +1,24 @@
+'''
+ @file      tally.py
+ @brief     Contains the Tally class
+ @author    Luke Eure
+ @date      October 13, 2015
+'''
+
+import numpy as np
+
+'''
+ @class Tally tally.py "tally.py"
+ @brief Tally contains tallies of information used in neutron simulation
+'''
+
+class Tally():
+    def __init__(self):
+        self.count = 0
+    def increment(self, amt):
+        self.count += amt
+    def clear(self):
+        self.count = 0
+    @property
+    def amt(self):
```

Comment:
Normally we call the attribute ``_count`` and we get the value using a function named ``count``. The ``amt`` name is a bit confusing.